### PR TITLE
[PLAT-823] Set up rss feed for cos.io

### PR DESCRIPTION
**Pull request by Johnetordoff**
## Purpose 

Our blog needs an RSS feed, Wagtail makes that easy, let's do it!

## Changes

- Adds BlogFeed class that automatically tranforms blogs into RSS.

## QA Notes 

Visit staging-cos.io/blog/feed/ and see that the RSS template has all the blog posts with accurate titles and descriptions.

## Tickets
https://openscience.atlassian.net/browse/PLAT-823

### CODE REVIEW

File: rss_feed/forms.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from django.contrib.syndication.views import Feed
+from blog.models import BlogPage
+
+
+class BlogsFeed(Feed):
+    title = "Cos.io blog"
+    link = "/blogs-rss/"
```
Comment (caseyrollins) [inline rss_feed/forms.py]: Does this mean that the RSS feed can be found at both `/blog/feed/` and `/blogs-rss/`? Or should this be the url that's registered in urls.py (i.e. `/blog/feed/`)?

File: website/settings/base.py [hunk 2]
```diff
@@ -68,6 +68,7 @@
 
     'wagtailfontawesome',
     'search',
+    'rss_feed',
```
Comment (caseyrollins) [inline website/settings/base.py]: Is this a separate app?

File: blog/urls.py [hunk 3]
```diff
@@ -1,8 +1,10 @@
 from django.conf.urls import url
+from rss_feed.forms import BlogsFeed
```
Comment (caseyrollins) [inline blog/urls.py]: Minor: why not add this class to blogs/views.py?

**Review by caseyrollins — COMMENTED** [review]
Just a few questions! Nice to see that this was so simple.

File: blog/urls.py (hunk 3 above)
Comment (Johnetordoff) [reply blog/urls.py]: That's probably I'll move this there.

File: rss_feed/forms.py (hunk 1 above)
Comment (Johnetordoff) [reply rss_feed/forms.py]: This link attribute is somewhat inexplicable, it appears in the RSS response and points back to the page that's sending it. The docs on this are pretty [vague](https://chrxr.com/adding-rss-feed-wagtail-site/)

**Pull request merged**